keps/sig-api-machinery/1929-built-in-default/README.md

```diff
@@ -276,12 +276,9 @@ endpoint today, but we have to decide what to do with the new default
 values of built-in types. The kube-openapi generated schemas will
 include them.
 
-We proppose to:
-1. leave CRDs not publishing defaults
-2. filter out defaults in the built-in type schemas too in a first step
-3. but possibly reconsider this behaviour in the future when the
-necessity of defaults for client-side merging (e.g. through kustomize)
-and influences on the ecosystem of default mis-use is better understood.
+We propose to:
+1. to allow the CRDs to publish the defaults field which helps the client to set the <zero-value>.
+2. publish out defaults in the built-in type schemas.
```

Comment:
You should probably have a paragraph that mentions that Kustomize and clients can't merge associative lists with default key fields if the default isn't visible, e.g. unable to merge port/protocol lists in the client.

Reply:
replaced the single line with the mentioned point.